core/pom.xml

```diff
@@ -26,6 +26,7 @@
 		<module>spin</module>
 		<module>client</module>
 		<module>storage</module>
+		<module>collection-factory</module>
```

Comment:
Should we put this new module under common, to reflect its general usage? Or if this is  scoped to be used internally in sail implementations only, it should perhaps be put under the sail module.

Reply:
I now think the API should be scoped to query evaluation/sails. With `@InternalUseOnly` set.